app/controllers/jobs_controller.rb

```diff
@@ -1,17 +1,21 @@
 class JobsController < ApplicationController
   before_action :set_job, only: [:update, :edit, :destroy]
   before_action :authenticate_user!, only: [:edit]
+  before_action :confirm_job_owner_is_current_user, only: [:edit, :destroy]
 
   def index
-    @jobs = Job.all
+    if user_signed_in?
+      @jobs = current_user.jobs
+    else
+    end
```

Comment:
this works 👍 

i would prefer that `@jobs` is always set even if it's empty, which might look like: 


```
@jobs = current_user&.jobs || []
```

This uses `&` ruby safe navigator which will only call `.jobs` if `current_user` is not `nil`